dual_net.py

```diff
@@ -329,12 +376,14 @@ def logging_ops():
 
 def compute_update_ratio(weight_tensors, before_weights, after_weights):
     """Compute the ratio of gradient norm to weight norm."""
-    deltas = [after - before for after, before in zip(after_weights, before_weights)]
+    deltas = [after - before for after,
+              before in zip(after_weights, before_weights)]
     delta_norms = [np.linalg.norm(d.ravel()) for d in deltas]
     weight_norms = [np.linalg.norm(w.ravel()) for w in before_weights]
     ratios = [d / w for d, w in zip(delta_norms, weight_norms)]
     all_summaries = [
-        tf.Summary.Value(tag='update_ratios/' + tensor.name, simple_value=ratio)
+        tf.Summary.Value(tag='update_ratios/' +
```

Comment:
splitting this line across the + is weird - which code autoformat are you using?

Reply:
autopep8